React/Modules/RCTAnimatedNode.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+
+@interface RCTAnimatedNode : NSObject
+
+- (instancetype)initWithTag:(NSNumber *)tag config:(NSDictionary *)config;
+
+@property (nonatomic, readonly) NSNumber *nodeTag;
+@property (nonatomic, strong) NSDictionary *config;
```

Comment:
Collection properties that have mutable subclasses (NSDictionary, NSArray, NSString) should be marked as copy instead of strong. Also please use lightweight generics to provide some indication of the collection contents (see below):

    @property (nonatomic, copy) NSDictionary<NSString *, id> *config;